my-app/src/pages/DiscoverAPI.js

```diff
@@ -0,0 +1,29 @@
+// getAllUsers
+export async function getAllUsers() {
+    const users = await fetch('https://disc-assignment-5-users-api.onrender.com/api/users', {
+      method: 'GET',
+    });
+    if (!users.ok) {
+      throw new Error('Failed to fetch users');
```

Comment:
using` try-catch ` blocks instead  of this if statement handles runtime errors better